src/PolykeyClient.ts

```diff
@@ -122,11 +240,14 @@ class PolykeyClient {
     this.logger.info(`Stopped ${this.constructor.name}`);
   }
 
-  public async destroy() {
+  public async destroy({ force = false }: { force?: boolean }) {
```

Comment:
I decided to make this default to `false`. I don't actually think `force` should be true by default. There is a special case in our transports that does this, but at an application level we should know why we are forcing it.

Reply:
@tegefaulkes @amydevs @addievo Still need to consider what happens for `PolykeyAgent`.